docs/features/sessions/pause.md

```diff
@@ -41,3 +41,29 @@ This is so that you don't lose your work!
 When you have added credits, just resume the session and carry on with your work. The left navigation bar always shows credits remaining.
 
 ![](/images/sessions/autopausing.gif)
+
+## Pause a Session Using a Github Action
+
+When you are done with your work day, you can setup Github Action to pause your session.  
+
+Below is scheduled-SessionPause.yml:
+
+```
+on:
+  schedule:
+    # Runs "at 8:25pm" everyday
```

Comment:
update this to contain the link to crontab - https://crontab.guru/
It will be useful for users to easily be able to update the crontab to meet their desired schedule

Reply:
removed the example